Resources/Locale/en-US/reagents/meta/medicine.ftl

```diff
@@ -147,3 +147,32 @@ reagent-desc-potassium-iodide = Will reduce the damaging effects of radiation by
 
 reagent-name-haloperidol = haloperidol
 reagent-desc-haloperidol = Removes most stimulating and hallucinogenic drugs. Reduces druggy effects and jitteriness. Causes drowsiness.
+
+# Moffstation Start - Additional holy books
+reagent-name-blessedbolognese = blessed bolognese
```

Comment:
I think a big new addition of localization like this should also go under a _moffstation directory.